src/api/public/apidocs-new/paths/search_channel_binary.yaml

```
@@ -0,0 +1,85 @@
+get:
+  summary: List channel objects that match a XPath condition.
+  description: |
+    Return a collection of channel objects with release informations that match a XPath condition.
```

```suggestion
    Return a collection of channel objects with release informations that match a XPath condition.
    
     This operation is the same as the one defined with `GET /search/channel/binary/id` with the exception of the results returned.
     While the former operation only returns a list of channel objects with the channel definition related to the binary, this one returns a list of channel
     objects with release information.
     See example values of a succeeded request below.
```
The reference to the `GET /search/channel/binary/id` was missing.